packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -167,27 +171,36 @@ const ProductFullDetail = props => {
         };
     }, [customAttributes, productDetails.sku, formatMessage]);
 
-    const cartCallToActionText = !isOutOfStock ? (
-        <FormattedMessage
-            id="productFullDetail.addItemToCart"
-            defaultMessage="Add to Cart"
-        />
-    ) : (
-        <FormattedMessage
-            id="productFullDetail.itemOutOfStock"
-            defaultMessage="Out of Stock"
-        />
-    );
-
+    const cartCallToActionText =
+        !isEverythingOutOfStock || !isOutOfStock ? (
+            <FormattedMessage
+                id="productFullDetail.addItemToCart"
+                defaultMessage="Add to Cart"
+            />
+        ) : (
+            <FormattedMessage
+                id="productFullDetail.itemOutOfStock"
+                defaultMessage="Out of Stock"
+            />
+        );
+    // Error message for screen reader
+    const outOfStockErrorMessage = 'This item is currently out of stock';
```

Comment:
Use `formatMessage()` to make this string translate-able

Reply:
Thanks! it's updated. Also added back the entry for this in `packages/venia-ui/i18n/en_US.json` and `packages/extensions/venia-sample-language-packs/i18n/fr_FR.json`